mycore-viewer/src/main/typescript/modules/iiif/widgets/IviewIIIFProvider.ts

```diff
@@ -37,7 +38,7 @@ namespace mycore.viewer.widgets.iiif {
                 url: manifestDocumentLocation,
                 success: (response: any) => {
                     const manifest = <Manifest> manifesto.create(response);
-                    const builder = new IIIFStructureBuilder(<Manifest> manifest, tilePathBuilder);
+                    const builder = new IIIFStructureBuilder(<Manifest> manifest, tilePathBuilder, imageAPIURL);
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ['IIIFStructureBuilder' is not defined. (no-undef)](https://app.codacy.com/app/MyCoRe/mycore/pullRequest?prid=4436103)